src/main/java/com/kodilla/ecommercee/domain/dto/OrderItemDto.java

```diff
@@ -0,0 +1,19 @@
+package com.kodilla.ecommercee.domain.dto;
+
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.Product;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class OrderItemDto {
+    private Long id;
+    private int quantity;
+    private Product product;
+    private Cart cart;
```

Comment:
Dto class should not contain full objects 
```suggestion
    private Long productId;
    private Long cartId;
```